ATL/schema.sql

```diff
@@ -0,0 +1,69 @@
+Create Table Passengers(
+Id int identity(1,1) Primary key,
+[Name] varchar(30),
+[Surname] varchar(50),
+Age int Not null,
+Check(Age>0)
+)
+
+ON
+
+Create Table AirLines(
+Id int identity(1,1) Primary key,
+[Name] varchar(50) not null,
+Concourse char not null

```

Comment:
With this structure we will have many different id's for the same Airline. How are we going to reference the Airline Delta from other tables if we can't uniquely identify it? Here you should have implemented a Many-to-Many relationship from Airline to a Concourse(different table).